src-java/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/service/session/Session.java

```diff
@@ -189,7 +189,7 @@ boolean handleResponse(OFMessage message) {
 
     private void actualWrite(OFMessage message)
             throws SwitchWriteException {
-        log.debug("push OF message to {}: {}", sw.getId(), message);
+        log.info("push OF message to {}: {}", sw.getId(), message);
```

Comment:
Doubt about this change. Will it repeat a log record for specific messages that already logged?

Reply:
It is not used to send packet-out for discovery requests (but used to send flow-ping). And we have the same log message of "old" way to send OF messages (https://github.com/telstra/open-kilda/blob/develop/src-java/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/switchmanager/SwitchManager.java#L2680).

I propose to give it a try and collect some usage experience. After that, we can decide how to improve it.

Reply:
ok